boards/arm/google_twinkie_v2/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# Google Twinkie V2 board configuration
+
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_GOOGLE_TWINKIE_V2
```

Comment:
Please split the board support and the sample application into separate commits.